staging/README.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+Welcome to the staging tree!
+This tree contains experimental code that does not (yet) meet the quality
+requirements of RIOT.
+
+**WARNING**
+
+Do not depend on anything within this subfolder!  Everything in here might be
+broken or break or be changed or removed at any time, and everything is
+completely unsupported! API's from this tree might change without notice.
+
+## How to use
+
+Please re-read the warning above.
+If you still feel brave enough to use code contained in here, add
+
+    ENABLE_STAGING=1
+
+to your application's Makefile.
+
+## Terminology
+
+"main tree" is used to describe the regular RIOT sources (everything outside
+staging/).
+
+"broken" means, it doesn't compile or work anymore.
+
+## Rules
+
+- no dependencies from main tree into staging are allowed
+- commits MUST NOT change files both in and outside of staging/
+- if anything in staging/ breaks due to a change in master, it will be marked
+  as broken
+- the original contributor is considered maintainer for a module
+- non-trivial changes SHOULD be ACK'ed by a module's maintainer
+
+## Procedures
+
+### Reviewing a change to staging/
+
+TBD
+
+### Mark as "broken"
+
+Modules can be marked as broken in staging/broken.mk.
+Add a comment with a date and a reason, e.g.,
+
+    BROKEN += heart   # 2018/12/24: broken through change in friend API (#12345)
+
```

Comment:
How does this work with changes to the main tree?

something like:
```
If the CI fails on a module from staging while developing for the main tree, prepend you sequence of commits with a commit as suggested above
```

Reply:
added